db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/check_pre_retirement.yaml

```diff
@@ -3,7 +3,7 @@ object_type: method
 version: 1.0
 object:
   attributes:
-    name: openstack_check_pre_retirement
+    name: check_pre_retirement
```

Comment:
@tinaafitz 
For pre_retirement can we move the method call check_pre_retirement into the method slot with multiple messages of create,openstack,microsoft and leave the openstack one slot empty. I am not sure which approach is better. If the same method is in a multi message field if a change needs to be done by the customer they would have to edit the class schema to change the message.